test/test_main.py

```diff
@@ -43,6 +50,15 @@ class SymlinkTargetError(Exception):
     pass
 
 
+class SimulatedEventSpec:
```

Comment:
I suggest s/Simulated//

`EventSpec` should be a fine name for this context, and will make the code below look slightly better.